apps/emqx_connector/src/emqx_connector_mongo.erl

```diff
@@ -178,12 +178,13 @@ health_check(PoolName) ->
 
 %% ===================================================================
 connect(Opts) ->
-    Type = proplists:get_value(mongo_type, Opts, single),
+    Type = proplists:get_value(type, Opts, single),
```

Comment:
@tigercl pls have a look at this change, things seem not be working without it 🤔

Reply:
related change: https://github.com/emqx/emqx/pull/6277/files

Reply:
Yes, I also made them, but after rebase only this one left.

Reply:
`mongo_type` should be used here, using `type` is a bug I introduced. I have fixed it in the PR mentioned by zmstone.

Reply:
@tigercl 

🤔 Here type is still set to `Opts`: https://github.com/emqx/emqx/pull/6277/files#diff-331b796db2f9509a1bf5d10eb34ec3c984fafd2c3f744466cfafde1bbc2fc945R121
And then these opts come to `connect/1` fun.

Reply:
@savonarola `mongo_type` is for the user layer, and `type` is for the driver layer.